### PR TITLE
feat: add control launch xml

### PR DESCRIPTION
**Pull request by wep21**
## Description

add control xml launch to simplify launch.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: launch/tier4_control_launch/launch/control.launch.xml [hunk 1]
```diff
@@ -0,0 +1,186 @@
+<launch>
```
Comment (kenji-miyake) [inline launch/tier4_control_launch/launch/control.launch.xml:1]: Will you keep both `.xml` and `.py` or remove `.py` later?
Anyway, I recommend creating a PR to delete `.py` because that enables you to track changes in `.py` easily.

Comment (kenji-miyake) [reply launch/tier4_control_launch/launch/control.launch.xml:1]: Also, it helps reviewers check the diff.

Comment (kenji-miyake) [reply launch/tier4_control_launch/launch/control.launch.xml:1]: Regarding the timing of `.py` deletion, it would be good to have several transition periods because it breaks the interface between `autoware_launch`.

Comment (wep21) [reply launch/tier4_control_launch/launch/control.launch.xml:1]: ! will remove python launch later because we also need to change [here](https://github.com/autowarefoundation/autoware_launch/blob/c6a3df8be00302a72f6fd2c447db7faf0d80475f/autoware_launch/launch/components/tier4_control_component.launch.xml#L11)

Comment (kenji-miyake) [reply launch/tier4_control_launch/launch/control.launch.xml:1]: Cool!

Comment (kenji-miyake) [reply launch/tier4_control_launch/launch/control.launch.xml:1]: https://github.com/autowarefoundation/autoware.universe/pull/2834

File: launch/tier4_control_launch/launch/control.launch.xml [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+        <param from="out/latest_external_control_cmd" to="/api/external/get/command/selected/control"/>
+        <param name="csv_path_accel_map" value="$(find-pkg-share raw_vehicle_cmd_converter)/data/default/accel_map.csv"/>
+        <param name="csv_path_brake_map" value="$(find-pkg-share raw_vehicle_cmd_converter)/data/default/brake_map.csv"/>
+        <param name="ref_vel_gain" value="3.0"/>
+        <param name="wait_for_first_topic" value="true"/>
+        <param name="control_command_timeout" value="1.0"/>
+        <param name="emergency_stop_timeout" value="3.0"/>
+        <extra_arg name="use_intra_process_comms" value="$(var use_intra_process)"/>
+      </composable_node>
+    </node_container>
```
Comment (kenji-miyake) [inline launch/tier4_control_launch/launch/control.launch.xml:171]: [Question] Why did you change here from `IncludeLaunchDescription`?
https://github.com/autowarefoundation/autoware.universe/blob/0324e054fdf1e2ff903aa6d3faa4e6752dc11b9d/launch/tier4_control_launch/launch/control.launch.py#L199-L223

File: launch/tier4_control_launch/launch/control.launch.xml [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+        <remap from="~/input/current_odometry" to="/localization/kinematic_state"/>
+        <remap from="~/input/current_steering" to="/vehicle/status/steering_status"/>
+        <remap from="~/input/current_accel" to="/localization/acceleration"/>
+        <remap from="~/input/current_operation_mode" to="/system/operation_mode/state"/>
+        <remap from="~/output/predicted_trajectory" to="lateral/predicted_trajectory"/>
+        <remap from="~/output/lateral_diagnostic" to="lateral/diagnostic"/>
+        <remap from="~/output/slope_angle" to="longitudinal/slope_angle"/>
+        <remap from="~/output/longitudinal_diagnostic" to="longitudinal/diagnostic"/>
+        <remap from="~/output/control_cmd" to="control_cmd"/>
+        <param from="$(var nearest_search_param_path)"/>
```
Comment (kenji-miyake) [inline launch/tier4_control_launch/launch/control.launch.xml:46]: Have you confirmed `param from` works for `composable_node`? (I assume so, but just in case.)
I remember there was a bug in Galactic, right?

**Review by kenji-miyake — COMMENTED** [review]
The diff looks good to me. If scenario tests pass, I think it's okay.

File: launch/tier4_control_launch/launch/control.launch.xml (hunk 2 above)
Comment (wep21) [reply launch/tier4_control_launch/launch/control.launch.xml:171]: Because it's more straightforward to write all the contents in xml, I think. There is only one `load_composable_nodes` in external_cmd_converter/selector python launch file.

File: launch/tier4_control_launch/launch/control.launch.xml (hunk 3 above)
Comment (wep21) [reply launch/tier4_control_launch/launch/control.launch.xml:46]: Yes, I confirmed [here](https://star4.slack.com/archives/C4P0NSMB5/p1659683096007659?thread_ts=1659679927.505959&cid=C4P0NSMB5). It's internal link, though.

Comment (kenji-miyake) [reply launch/tier4_control_launch/launch/control.launch.xml:46]: Thank you, great!

For others, I'll paste the content below.

```console
❯ cat test.launch.xml
<launch>
  <node_container pkg="rclcpp_components" exec="component_container" name="my_container" namespace="" args="test_args">
    <composable_node pkg="demo_nodes_cpp" plugin="demo_nodes_cpp::ParameterBlackboard" name="board1" namespace="test_namespace">
      <param from="/home/daisuke/test.yaml" />
    </composable_node>
  </node_container>

  <load_composable_node target="my_container">
    <composable_node pkg="demo_nodes_cpp" plugin="demo_nodes_cpp::ParameterBlackboard" name="board2" namespace="test_namespace">
      <param from="/home/daisuke/test.yaml" />
    </composable_node>
  </load_composable_node>
</launch>
```

```console
❯ cat test.yaml
/**:
  ros__parameters:
    param1: aaa
    param2: 1
    param3: 2.0
```

```console
❯ ros2 param dump /test_namespace/board1
/test_namespace/board1:
  ros__parameters:
    param1: aaa
    param2: 1
    param3: 2.0
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    use_sim_time: false


~/workspace/WebAutoMLEvaluations develop*
❯ ros2 param dump /test_namespace/board2
/test_namespace/board2:
  ros__parameters:
    param1: aaa
    param2: 1
    param3: 2.0
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    use_sim_time: false
```

File: launch/tier4_control_launch/launch/control.launch.xml (hunk 2 above)
Comment (kenji-miyake) [reply launch/tier4_control_launch/launch/control.launch.xml:171]: I think it's okay to replace it here. (It would be ideal to do it in another PR, though.)
Then, don't you need to remove the old `.py` files of external_cmd_converter/selector?
